index.css

```diff
@@ -93,6 +95,10 @@ ul {
 a {
   text-decoration: none;
   color: inherit;
+  cursor: pointer;
+}
+.active {
+  color: var(--action-color);
 }
 .btn {
   padding: 0.5em;
```

Comment:
i would suggest you to use "rems" throughtout

Reply:
I only used ems here to scale with the font size as em when used for lengths depend on the font size

> i would suggest you to use "rems" throughtout